### PR TITLE
Fix for Issue 'English app download buttons on German page #2431'

### PR DESCRIPTION
**Pull request by Crystalskulls**
- Fix for Issue 'English app download buttons on German page Issue #2431'
Added german google and apple svg badges and renamed bad translated label 'App mit!' to 'App herunterladen!'

---
Internal Tracking ID: [EXPOSUREAPP-11892](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-11892)

### CODE REVIEW

File: src/data/index_de.json [hunk 1]
```diff
@@ -352,19 +352,19 @@
             "icon-list": [
                 {
                     "url": "https://apps.apple.com/de/app/corona-warn-app/id1512595757",
-                    "icon-url": "/assets/img/App_Store_Badge_US-UK.svg",
+                    "icon-url": "/assets/img/App_Store_Badge_DE.svg",
                     "title": "Laden sie die Corona-Warn-App aus dem Apple App Store",
```
Comment (MikeMcC399) [inline src/data/index_de.json:356]:
```suggestion
                    "title": "Laden Sie die Corona-Warn-App aus dem Apple App Store",
```

File: src/data/index_de.json [hunk 2]
```diff
@@ -352,19 +352,19 @@
...
                     "external": true
                 },
                 {
                     "url": "https://play.google.com/store/apps/details?id=de.rki.coronawarnapp",
-                    "icon-url": "/assets/img/google-play-badge.svg",
+                    "icon-url": "/assets/img/google-play-badge-DE.svg",
                     "title": "Laden sie die Corona-Warn-App aus dem Google Play Store",
```
Comment (MikeMcC399) [inline src/data/index_de.json:362]:
```suggestion
                    "title": "Laden Sie die Corona-Warn-App aus dem Google Play Store",
```

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
Added comment to also correct grammar in previously existing text.

**Comment by dsarkar** [comment]
@Crystalskulls Thanks for the PR! Please see comments from @MikeMcC399 above!

**Comment by Crystalskulls** [comment]
@dsarkar I submitted the suggestion from MikeMcC399

**Review by Crystalskulls — COMMENTED** [review]
Files reviewed

**Review by MikeMcC399 — APPROVED** [review]
LGTM!

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Crystalskulls @MikeMcC399 Thanks!

**Pull request merged**